tests/e2e/wilcoEngine/utils.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require("@jest/globals");
+const { sleep } = require("../../utils");
+const { subscribe, ubsubscribe } = require("./wilcoEngineEvents");
+
+const execAndWaitForEvent = async (type, func, maxTime = 500) => {
```

Comment:
Maybe `event_type`?

Reply:
Since when do we use snake_case? Will call it eventType

Reply:
Right, just some python leftovers...